tools/osx/xcode_locator.m

```diff
@@ -177,8 +177,16 @@ static void AddEntryToDictionary(
           url, version, expandedVersion);
 
     NSURL *versionPlistUrl = [url URLByAppendingPathComponent:@"Contents/version.plist"];
-    NSDictionary *versionPlistContents = [[NSDictionary alloc] initWithContentsOfURL:versionPlistUrl
-                                                                               error:nil];
+
+    // macOS 10.13 changed the signature of initWithContentsOfURL,
+    // and deprecated the old one.
+    NSDictionary *versionPlistContents;
+#if MAC_OS_X_VERSION_MIN_REQUIRED > MAC_OS_X_VERSION_10_12
+    versionPlistContents = [[NSDictionary alloc] initWithContentsOfURL:versionPlistUrl                                                                                                                   error:nil];
```

Comment:
Is this missing `error:`?

Reply:
It wasn't, but the formatting was messed up (I did test both sides of the condition). Fixed, thanks.